### PR TITLE
doc: add future versions 21-24 to release schedule

### PR DESCRIPTION
**Pull request by targos**

### CODE REVIEW

File: schedule.svg
Comment (targos) [inline schedule.svg:1]: I see that the first line changed from main to master, but I just followed the docs to generate the SVG.

**Review by UlisesGascon — APPROVED** [review]

File: schedule.svg
Comment (richardlau) [reply schedule.svg:1]: It was manually updated from "master" to "main" in the SVG in https://github.com/nodejs/Release/pull/775#pullrequestreview-1080801641. 

We'd need https://github.com/nodejs/lts-schedule/pull/15 in the lts module.

File: schedule.json [hunk 1]
```diff
@@ -113,5 +113,29 @@
...
     "end": "2026-04-30",
     "codename": ""
+  },
+  "v21": {
+    "start": "2023-10-17",
+    "maintenance": "2024-04-01",
+    "end": "2024-06-01"
+  },
+  "v22": {
+    "start": "2024-04-23",
```
Comment (richardlau) [inline schedule.json:123]: I think we usually go for the third Tuesday in the month, although I'm not sure that's written down as policy anywhere.

File: schedule.svg
Comment (targos) [reply schedule.svg:1]: Manually fixed again

File: schedule.json (hunk 1 above)
Comment (targos) [reply schedule.json:123]: Does it matter? I counted from the other side (second to last Tuesday in the month)

**Review by richardlau — APPROVED** [review]

**Review by RafaelGSS — APPROVED** [review]

File: schedule.svg
Comment (richardburce19) [inline schedule.svg:1]: @targos

**Review by richardburce19 — COMMENTED** [review]
#912

**Comment by yehiamohamedyehia281** [comment]
Thank you

**Pull request merged**